mysql/src/test/scala/zio/sql/mysql/FunctionDefSpec.scala

```diff
@@ -148,6 +148,17 @@ object FunctionDefSpec extends MysqlRunnableSpec with ShopSchema {
 
       assertZIO(testResult.runHead.some)(equalTo(expected))
     },
+    test("rand") {
+      val query = select(Rand(5))
+
+      val testResult = execute(query)
+
+      val assertion = for {
+        r <- testResult.runCollect
+      } yield assert(r.head)(Assertion.isGreaterThanEqualTo(0d) && Assertion.isLessThanEqualTo(1d))
+
+      assertion.mapErrorCause(cause => Cause.stackless(cause.untraced))
```

Comment:
We've changed the convention of repeated block to something shorter. Please take a look at this PR https://github.com/zio/zio-sql/pull/687/files and adjust for the same convention :pray:

Reply:
done!